packages/quantic/server.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
+MIIEpAIBAAKCAQEAudtPp53b81MLispdZgdNXbjVOneqqF5eibpDY3DVYCaoklaR
```

Comment:
this should be removed immediately. The secret is stored in the github repo and should not be shown in clear text here or anywhere that is not your local setup or lastpass.

Reply:
As a precaution, please add an entry in the `.gitignore` file just to avoid this issue in the future.

Reply:
Absolutely, sorry about this

Reply:
FYI, I'm not sure what this key is for, but it should be also revoked from the original system (if it's important...)

You can be sure it's on a ton of various databases by now ;)

Reply:
Yes it's been deactivated :)